client/lib/plans/constants.js

```diff
@@ -546,6 +547,28 @@ export const PLANS_LIST = {
 		getPathSlug: () => 'premium-2-years',
 	},
 
+	[ PLAN_BUSINESS_MONTHLY ]: {
+		...getPlanBusinessDetails(),
+		term: TERM_MONTHLY,
+		getBillingTimeFrame: i18n.translate( '/month, billed monthly' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'per month, billed monthly' )` *ES Score: 20*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/feature/monthly-business-plan)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).